### ISSUE TITLE
New mod request: diogo-gomes/mob-engine (fork of abandoned BlockMen/cme)

### ISSUE THREAD

**Issue by diogo-gomes** [minetest-mods/minetest-mods.github.io#22]
Hello, I want to request a fork of an abandoned mod.

My repository is diogo-gomes/mob-engine

Apparently, the developer abandoned the development of everything related to Minestest and might not want to be contacted.

Can you help?

Thank you

**Comment by sofar** [comment]
The guidelines are that the original author needs to request the mod to be moved to minetest-mods, since otherwise everyone could just fork it do exactly the same thing. We therefore do not accept forks in general.

CME has been forked several times as well, so I'm not inclined to accept one of the forks without some larger group discussion where many fork maintainers agree collaboratively.

Let me rephrase: if I get a response from several fork maintainers that they want to collaboratively maintain this code, then I'd be willing to accept.

**Comment by diogo-gomes** [comment]
Thanks for the answer.

As you can see in the picture, there are only eight forks and all forks are quite outdated compared to Blockmen's repo.

But I enjoyed your suggestion in relation to a discussion about the future of the mod.

I'll try to get in touch with the developers of forks and promote this discussion.

Thank you again.

(If you want, you can close this issue until there).

![captura de ecra de 2016-10-30 18-16-32](https://cloud.githubusercontent.com/assets/9022734/19838735/46206012-9ecd-11e6-89a4-c09dca8535e2.png)

**Comment by sofar** [comment]
I'm interested to see if your method of approach works or not. Please update this issue yourself when you have anything to share. I'll leave it open for now.

**Comment by diogo-gomes** [comment]
# Updates
## Answers

| Total | 4 | Total | 4 |
| --- | --: | --- | --: |
| Agree | 4 | Maintain | 3 |
| Not agree | 0 | Not maintain | 1 |

Updated at 31/10/2016

---
More information in this [link](https://github.com/BlockMen/cme/issues/12).

**Comment by sofar** [comment]
Without reading the entire debate, the data you posted in the last update suggests that there are 4 people willing to help maintain a minetest-mods branch of CME, is that correct? Who are they?

**Comment by diogo-gomes** [comment]
Yes, correct. They are:

- @BrunoMine
- @diogo-gomes (me)
- @LNJ2
- @mahmutelmas06

**Comment by LNJ2** [comment]
@diogo-gomes Sorry, I probably won't help to maintain it anymore (I'm doing other things currently). Maybe some bug-fixes.
FYI: If you want to use any changes I made in the TNG version of the CME, they're also available under the original license. (See the subfolders of https://github.com/LNJ2/Minetest_TNG/tree/master/mods/cme)

**Comment by diogo-gomes** [comment]
@LNJ2 It's okay. I'll see your changes. Thanks.

**Comment by sofar** [comment]
What is the code base for the continuation for this mod? URL? License?

**Comment by diogo-gomes** [comment]
**[Original repository](https://github.com/BlockMen/cme)**

**[My repository (future)](https://github.com/diogo-gomes/mob-engine)**

**Url:** mob-engine

**Actual license**
Code:  (c) Copyright 2015-2016 BlockMen; modified zlib-License
Media: (c) Copyright (2014-2016) BlockMen; CC-BY-SA 3.0

**Future license:** Undecided

**Comment by LNJ2** [comment]
I would suggest LGPLv2.1+ as Minetest, zlib is compatible with LGPL (also the modified version?)

**Comment by sofar** [comment]
The zlib license listed does not allow relicensing as far as I can see.

**Comment by sofar** [comment]
approved, under condition that the modified zlib-license stays.

**Comment by sofar** [comment]
I've granted permissions to
@BrunoMine
@diogo-gomes (me)
@mahmutelmas06

**Comment by sofar** [comment]
https://github.com/minetest-mods/mob-engine

unforked clone, ready for development. Good luck!

**Comment by diogo-gomes** [comment]
@sofar,

Thank you for the approval.

About the license and after research, I open a question in [Open Source beta (StackExchange)](http://opensource.stackexchange.com/questions/5295/can-i-relicense-software-under-the-modified-zlib-licence).

Who have permissions to manage branches, wiki, issues, etc?
